FIX Standard/OrchestraFIX44.xml

```diff
@@ -40949,7 +40949,7 @@
 				<fixr:groupRef id="1014" presence="required" added="FIX.4.4">
 					<fixr:annotation>
 						<fixr:documentation>
-         Insert here here the set of "Position Amount Data" fields defined in "Common Components of Application Messages"
+         The set of "Position Amount Data" fields defined in "Common Components of Application Messages"
```

Comment:
FIX Latest has over 200 places where we use the text "Insert here..." and we do not want to change that. Agree to remove the duplication "here here..."